ci-operator/step-registry/redhat-chaos/hog-scenarios/OWNERS

```diff
@@ -0,0 +1,8 @@
+approvers:
+  - chaitanyaenr
```

Comment:
We should update this file to be a symbolic link to the owners file at the main redhat-chaos folder like https://github.com/openshift/release/blob/master/ci-operator/step-registry/redhat-chaos/pvc-scenario/OWNERS

Reply:
doing this type of set up we dont need to edit each subfolders owners file when we add someone new (like Sahil)

Reply:
Agreed! Updated the PR with the changes. Thanks.